src/config/machines/transition-patterns.js

```diff
@@ -0,0 +1,20 @@
+// src/config/machines/transition-patterns.js
+
+/**
+ * Creates standard transitions for a page in the journey
+ * @param {string} backTarget - The state to go back to
+ * @param {string} nextTarget - The state to go forward to
+ * @returns {object} Standard transition configuration
+ */
+export const createTransitions = (backTarget, nextTarget) => ({
```

Comment:
Great idea to extract it to separate file 👍 
Could we improve this to change the params to an object like so:
`/**
 * Creates standard transitions for a page in the journey
 * @param {object} targets - Object containing target states
 * @param {string} targets.back - The state to go back to
 * @param {string} targets.next - The state to go forward to
 * @returns {object} Standard transition configuration
 */
export const createTransitions = ({ back, next }) => ({
  on: {
    BACK: {
      target: back,
      actions: ['updateCurrentPageId']
    },
    NEXT: {
      target: next,
      actions: ['trackPageCompletion', 'updateCurrentPageId', 'updateAnswers']
    }
  }
});`
and then usage:
`country: {
  ...createTransitions({ back: 'start', next: 'second-question' }),
  meta: {
    ...QUESTION_META,
    currentPageId: 'country',
    previousPageId: 'start',
    nextPageId: 'second-question',
    // other meta properties
  }
},
`
WDYT?

Reply:
I've done a redo based on David's suggestions and yours. See if it works for you.

Reply:
Does this not assume that all transitions are back and next? Maybe I'm missing something

Reply:
Yeah I was thinking about that too but once we introduce new types of transitions we will improve?

Reply:
In fact I have some improvement in my coming PR to get rid of the DRY around `meta.nextPageId` and `on.NEXT.target` so I may as well address the different types of transition there too.